src/wp-includes/class.wp-styles.php

```diff
@@ -278,6 +278,12 @@ public function do_item( $handle, $group = false ) {
 			}
 		}
 
+		if ( $obj->async && ( false === strpos( $tag, "media='print'" ) ) ) {
+			$async_tag  = '<noscript>' . str_replace( " id='", " id='no-js-fallback-", $tag ) . '</noscript>';
+			$async_tag .= str_replace( " media='{$media}'", " media='print' onload='this.media=\"all\"'", $tag );
```

Comment:
```suggestion
			$async_tag .= str_replace( " media='{$media}'", " media='print' onload='this.media=\"{$media}\"'", $tag );
```

Shouldn't this be this?

Reply:
Yes it should, thank you!